template/docs/_static/css/custom.css

```diff
@@ -13,9 +13,28 @@ html[data-theme="dark"] {
  */
 .bd-header .navbar-nav li a.nav-link .icon-link-image {
     filter: var(--scipp-filter-header-icon);
+    height: 1em;
 }
 
 .bd-header .navbar-nav li a.nav-link .icon-link-image:hover {
     /* match primary color */
-    filter: hue-rotate(85deg) saturate(0.829) brightness(0.945);
+    filter: hue-rotate(153deg) saturate(1.28) brightness(0.7);
```

Comment:
What was the reason for changing the numbers here?

Reply:
The theme changed the link color from the old blue to purple. (`pst-link-color` instead of `--pst-primary-color`). I need to fix the comment

Reply:
I updated the colours again using https://codepen.io/sosuke/pen/Pjoqqp. They should be more accurate now.

Also, I changed the CSS selector to also work with a side menu on a very narrow screen: 

![Screenshot_20231113_111703](https://github.com/scipp/copier_template/assets/11393224/cd296a74-f080-4b40-af3d-1a9041d98a06)